extensions/ql-vscode/src/variant-analysis/ql.ts

```diff
@@ -0,0 +1,87 @@
+import { dirname } from "path";
```

Comment:
Is there a more descriptive name we can use for this file? Unfortunately I can't think of any better examples, but I find just "ql" a bit unclear. It also isn't helped that there's `extensions/ql-vscode/src/common/ql.ts` too which this could get confused with.

Reply:
`ql.ts` made sense as it's QL-related code for MRVA. I couldn't think of a better name. We could prefix it if you think it helps? e.g. `variant-analysis-ql.ts`?

BTW I can imagine other functions being moved here (and out of run-remote-query.ts).